### PR TITLE
Pass tokens as params for automatic login

### PR DESCRIPTION
**Pull request by luiscuenca**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19607/HighFidelity-Beta-PR15618-206e353.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19607/HighFidelity-Quest-Beta-PR15618-206e353.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19607/HighFidelity-Beta-PR15618-206e353.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19607/HighFidelity-Beta-Interface-PR15618-206e353.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19607/HighFidelity-Beta-PR15618-206e353.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19607/HighFidelity-Beta-Interface-PR15618-206e353.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19610/HighFidelity-Beta-PR15618-6d824ff.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19610/HighFidelity-Beta-Interface-PR15618-6d824ff.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19610/HighFidelity-Beta-PR15618-6d824ff.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19610/HighFidelity-Quest-Beta-PR15618-6d824ff.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19610/HighFidelity-Beta-PR15618-6d824ff.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19610/HighFidelity-Beta-Interface-PR15618-6d824ff.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19617/HighFidelity-Beta-PR15618-8db3d03.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19617/HighFidelity-Quest-Beta-PR15618-8db3d03.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19617/HighFidelity-Beta-PR15618-8db3d03.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19617/HighFidelity-Beta-Interface-PR15618-8db3d03.exe

**Comment by luiscuenca** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19626/HighFidelity-Beta-PR15618-8db3d03.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19626/HighFidelity-Quest-Beta-PR15618-8db3d03.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19626/HighFidelity-Beta-PR15618-8db3d03.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19626/HighFidelity-Beta-Interface-PR15618-8db3d03.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19626/HighFidelity-Beta-PR15618-8db3d03.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19626/HighFidelity-Beta-Interface-PR15618-8db3d03.exe

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -9355,6 +9355,16 @@ void Application::showUrlHandler(const QUrl& url) {
     });
 }
 
+void Application::forceDisplayName(const QString& displayName) {
+    getMyAvatar()->setDisplayName(displayName);
+}
+void Application::forceLogginWithTokens(const QString& tokens) {
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: spelling mistake `forceLogginWithTokens` should be `forceLoginWithTokens`

File: interface/src/main.cpp [hunk 2]
```diff
@@ -398,6 +405,21 @@ int main(int argc, const char* argv[]) {
...
+            if (parser.isSet(displayNameOption)) {
+                QString displayName = QString(parser.value(displayNameOption));
+                appPointer->forceDisplayName(displayName);
+            }
+            if (!launcherPath.isEmpty()) {
+                appPointer->setConfigFileURL(configFileName);
+            }
+            if (parser.isSet(responseTokensOption)) {
+                QString tokens = QString(parser.value(responseTokensOption));
+                appPointer->forceLogginWithTokens(tokens);
```
Comment (ZappoMan) [inline interface/src/main.cpp]: typo

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19639/HighFidelity-Beta-PR15618-42c1f4b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19639/HighFidelity-Beta-Interface-PR15618-42c1f4b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19639/HighFidelity-Beta-PR15618-42c1f4b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19639/HighFidelity-Quest-Beta-PR15618-42c1f4b.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19639/HighFidelity-Beta-PR15618-42c1f4b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19639/HighFidelity-Beta-Interface-PR15618-42c1f4b.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19641/HighFidelity-Beta-PR15618-050682c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19641/HighFidelity-Quest-Beta-PR15618-050682c.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19641/HighFidelity-Beta-PR15618-050682c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19641/HighFidelity-Beta-Interface-PR15618-050682c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19641/HighFidelity-Beta-PR15618-050682c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19641/HighFidelity-Beta-Interface-PR15618-050682c.dmg

**Pull request merged**